features/machine/machine_misc.feature

```diff
@@ -45,16 +45,18 @@ Feature: Machine misc features testing
      | n | openshift-machine-api |
    And admin ensures "mypod" pod is deleted after scenario
 
-   Then I get project events
-   And the output should match:
-     | Failed to provision volume with StorageClass "thin": Credentials not found|
+   Given version dependent csi feature
+   Then the step should succeed
 
    Then I run the :replace admin command with:
       | _tool | oc                           |
       | f     | vsphere-creds_original.yaml  |
       | force |                              |
    And the step should succeed
 
+   Given I use the "openshift-machine-api" project
+   When 300 seconds have passed
```

Comment:
@miyadav what's the purpose of the hard-coded sleep?  We discourage hardcoing the sleep time, so try to avoid it if possible.

Reply:
The password sync takes time , safer option is to have it hard-coded, as we do not want it to wait too long as well since those situations will be when cluster becomes unstable .

Reply:
Hi Milind, can you share what is this waiting for? We can use wait_for instead of a hard wait.

Reply:
@jhou1 , I made it `wait upto 300 sec `now , I hope it is fine now , basically when we revert the password in vsphere-creds it takes variable amount of time , not more than 5 mins , will just keep checking for the log as you suggested.

Reply:
changes looks good for 4.13 took 9m approx - https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/750955/console

4.12 took 6m approx - https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/750956/console